src/sentry/models/integration.py

```diff
@@ -66,11 +66,15 @@ class Meta:
 class ExternalProviders(Enum):
     GITHUB = 0
     GITLAB = 1
+    EMAIL = 2
```

Comment:
I think we should do the same multiples of ten thing here to help keep the providers organized by type. For example 
```
EMAIL = 100
SLACK = 110
```

Reply:
@mgaeta I was thinking that as I added them but I didn't want to touch the existing stuff and thought it might be weird to jump ahead in a huge number, but if you think it's fine I'll make that change